http_client.rst

```diff
@@ -330,6 +330,10 @@ If you use for example
 as the type and name of an argument, autowiring will inject the ``my_api.client``
 service into your autowired classes.
 
+.. note::
+
+    Pay attention to the trailing slashes in the `base_uri` parameter and prefixed slashes in your path when making a request. To keep the path in your `base_uri` it must end with a slash and your request path must NOT start with a slash. E.g. If you make a request to the path `endpoint` with the `base_uri` `https://example.com/api/` the request will be sent to `https://example.com/api/endpoint`. If you make a request to the path `/endpoint`, or your `base_uri` is `https://example.com/api`, the request will be sent to `https://example.com/endpoint`
```

Comment:
In the code example above, we already have this comment:

> using base_url, relative URLs (e.g. request("GET", "/repos/symfony/symfony-docs")) will default to these options

Do you think we could reword that a bit to make the behavior more clear?

Reply:
I was trying to work out how to make this simple enough to use in that comment but wasn't sure how. I think the comment shouldn't use the prefixes slash though as it'd be an example that'd override the path in the base_uri if defined.

Reply:
I think the note could just give the link to https://symfony.com/doc/current/reference/configuration/framework.html#base-uri as a reference and say pay attention to slashes. Perhaps I should have found that anyway, but it'd be good just for clarity for those others who may have missed that like I did.

Reply:
I've attempted new wording for this tooltip